ui.frontend/src/view/FormPanel.js

```diff
@@ -40,15 +40,36 @@ export default class FormPanel extends FormFieldBase {
                 instanceManager = this.instantiateInstanceManager();
                 this.formContainer.addInstanceManager(instanceManager);
             }
-            instanceManager.addChild(this);
             this.setInstanceManager(instanceManager);
+            instanceManager.addChild(this);
         }
     }
 
     addChild(childView) {
         this.children.push(childView);
     }
 
+    getChild(id){
+        for(let key in this.children){
+            if(this.children[key].id===id){
+                return this.children[key];
+            }
+        }
+    }
+
+    customRepeatableHtmlHandler(instanceManager,addedModel,htmlElement,beforeElement){
+        var instanceIndex=addedModel.index;
+        if (instanceManager.children.length == 0) {
+            instanceManager.parentElement.append(htmlElement);
```

Comment:
as instance manager is doing the stuff, encapsulate this function in instance manager and invoke that

Reply:
done